packages/react-vapor/src/components/optionsCycle/OptionsCycle.tsx

```diff
@@ -74,7 +74,7 @@ export class OptionsCycle extends React.Component<IOptionsCycleProps> {
                     onClick={() => this.goToPreviousOption()}
                     disabled={!this.props.wrapAround && this.props.currentOption === 0}
                 >
-                    <Svg svgName="arrow-left-rounded" svgClass="icon mod-16" />
+                    <Svg svgName="arrow-left-rounded" className="icon mod-16" />

```

Comment:
Is this still required? IIRC `svgClass` is more appropriate as it will set those classes on the `svg` tab instead of the wrapping `span`.

Reply:
The problem is that the `span` (which is vertically centered) doesn't take the full height of the `svg`. For this simple fix I didn't want to change the alignment of the SVG because I didn't want to impact all of react-vapor